modules/gitops-setting-up-the-argocd-agent.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assembly:
+//
+// * argocd-agent-installation/gitops-setting-up-the-argocd-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-setting-up-the-argocd-agent_{context}"]
+= Setting up the Argo CD Agent
+
+After you set up the Principal component, you can connect it with one or more Agents.
+
+.Prerequisites
+
+* The Principal setup is complete and running.
+* You have access to both the Principal and Agent clusters.
+* The `argocd-agentctl` CLI tool is installed and accessible from your environment.
+* The Helm CLI is installed and configured.
+
+.Procedure
+
+. Create the Agent secret on the Principal cluster.
++
+To create a secret on the Principal cluster that contains configuration details for the Agent, run the following command:
++
+[source,terminal]
+----
+$ argocd-agentctl agent create "<AGENT_NAME>" \
+  --principal-context "<PRINCIPAL_CONTEXT>" \
+  --principal-namespace "<PRINCIPAL_NAMESPACE>" \
+  --resource-proxy-server "<PRINCIPAL_RESOURCE_PROXY_URL>"
+----
++
+where:
++
+`<AGENT_NAME>`:: Assigns a unique name to identify each agent managed by the principal.
+`<resource-proxy-server>`:: Specifies the address of the Principal's `resource-proxy` service that the Agent connects to.
```

Comment:
This is incorrect: Specifies the address of the Principal's `resource-proxy` service ~that the Agent connects to~.

Jayendra can correct me if wrong, but the suggestion we can provide to the user for this field is that the value should be `(.metadata.name field from ArgoCD CR)-agent-principal-resource-proxy:9090`.